test/e2e/configurer/chain/queries.go

```diff
@@ -180,7 +180,9 @@ func (n *NodeConfig) QueryListSnapshots() ([]*cmtabcitypes.Snapshot, error) {
 func (n *NodeConfig) QueryRawCheckpoint(epoch uint64) (*ct.RawCheckpointWithMetaResponse, error) {
 	path := fmt.Sprintf("babylon/checkpointing/v1/raw_checkpoint/%d", epoch)
 	bz, err := n.QueryGRPCGateway(path, url.Values{})
-	require.NoError(n.t, err)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	require.NoError(n.t, err)
```
Failing to query the node should not happen so we can panick here